config/crd/bases/csi-baremetal.dell.com_deployments.yaml

```diff
@@ -0,0 +1,327 @@
+
```

Comment:
We store CRD in charts folder.

Reply:
https://github.com/dell/csi-baremetal-operator/blob/master/Makefile#L74  

>  output:crd:dir=config/crd

Reply:
I've not changed any settings. According to `Makefile` the output of crds is the config folder.